api/api.js

```diff
@@ -0,0 +1,18 @@
+import fetch from 'isomorphic-fetch';
+
+
+const api = {
```

Comment:
Do we have a reason to export the whole object instead of functions itself? Also the exported name could be more clear:

Something like:
```
const baseURL = 'https://pereiratechtalks.com'

export async getPTTPosts () {
  ....
}

```